docs/index.html.ejs

```diff
@@ -965,6 +965,35 @@
             });
           </script>
       `) %>
+    </div>
+    </section>
+
+    <section class="component">
+      <h3 id="progress-indicator">Progress Indicator</h3>
+      <div>
+        <blockquote>
+          You can use a <em>progress indicator</em>, also known as a <em>progress bar control</em>, to show the percentage of completion of a lengthy operation.
+
+          <footer>
+            &mdash; Microsoft Windows User Experience p. 189
+          </footer>
+        </blockquote>
+
+        <p>
+          There are two types of progress bars: solid and segmented. The solid version is the default. To declare a segmented bar, you should use the <code>segmented</code> class.
+        </p>
+
+        <%- example(`
+          <div class="progress-indicator">
+            <span class="progress-indicator-bar"  style="width: 400px;" />
```

Comment:
Maybe it would be more semantic to do something like `width: 40%;`? Or is there a downside to using a percentage here?

Reply:
That actually makes far more sense. Changed in the last diff!